javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-player-app/script.js

```diff
@@ -5,97 +5,182 @@ const nextButton = document.getElementById("next");
 const previousButton = document.getElementById("previous");
 const shuffleButton = document.getElementById("shuffle");
 
+// Short mp3s for testing
```

Comment:
since this look like it was only used for testing locally, I think we should revert the allSongs list back to its original state

Reply:
Thanks for bringing this up @jdwilkin4.

I pushed a commit to have these test mp3s commented out by default.

Reply:
What if campers want to test this themselves as they are building the project? I think we should leave in a few very short mp3s so that they can easily do this.

Reply:
A minor thing: The dashed outline stays put and does not follow to the next song. It's unsynced with song playing.

Reply:
Thanks for this helpful suggestion, @bbsmooth.

I was working on this project with @RafaelDavisH today and he offered to create some short 10 second MP3 files we can use for some steps in this project.

I'm thinking that, for the steps where learners will write code to play the next song once the current song is finished, we can update the `allSongs` array in the seed for them with the shorter files for testing, and mention it in the description. Once learners are finished coding and testing that feature, we can revert the seed back to the original `allSongs` array with the longer tracks.